org.kde.ark.json

```diff
@@ -0,0 +1,54 @@
+{
+    "id": "org.kde.ark",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "ark",
+    "rename-icon": "ark",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
Is filesystem=host really needed while this app should work just fine with portals?

Reply:
AFAIK the portal is just "give me one file to work with", no?

That's fine for opening an archive, but how is that going to unarchive something?

Reply:
You are right, portals still suck.

Reply:
This could potentially be removed but would require some changes in the App. I don't think this is a priority right now.